### PR TITLE
Master

### PR DESCRIPTION
**Pull request by albz**
minor error in the weighting for the background diagnostic

### CODE REVIEW

File: src/read_input.f90 [hunk 1]
```diff
@@ -580,8 +580,8 @@ end subroutine read_input_data
  subroutine consistency_check_number_of_particles_comp
  if(all(ppc>=1)) then
   call from_ppc_to_npx_npy_npz
- else
-  np_per_zc=np_per_yc
+ elseif(all(np_per_zc)==-1) then
```
Comment (cenit) [inline src/read_input.f90]: Our master is already with this fix, not sure why it is highlighted here...

File: src/util.f90 [hunk 2]
```diff
@@ -820,7 +820,7 @@ subroutine generate_bunch_bigaussian_weighted( &
  real(dp), allocatable :: ppcb_positions(:,:)
 
  bch=weight
- sigma_cut=3.0
+ sigma_cut=4.0
```
Comment (cenit) [inline src/util.f90]: It looks like you often need to change sigma_cut. Why not move it into an input parameter?

File: src/pdf_moments.f90 [hunk 3]
```diff
@@ -1288,7 +1288,7 @@ SUBROUTINE diagnostic_integrated_background
  do ip=1,np_local
         if(mask(ip)) then
                wgh=spec(1)%part(ip,7)
-                mu_gamma_local  = mu_gamma_local + sqrt(   1.0 + spec(1)%part(ip,4)**2 + &
+                mu_gamma_local  = mu_gamma_local + sqrt(1.0 + spec(1)%part(ip,4)**2 + &
```
Comment (cenit) [inline src/pdf_moments.f90]: No problems soved here, just spacing. In the merge the headline will be changed

File: src/pdf_moments.f90 [hunk 4]
```diff
@@ -1301,9 +1301,10 @@ SUBROUTINE diagnostic_integrated_background
  do ip=1,np_local
         if(mask(ip)) then
                wgh=spec(1)%part(ip,7)
-               s_gamma_local  = s_gamma_local+ (sqrt(1.0 + spec(1)%part(ip,4)**2 + &
+               s_gamma_local  = s_gamma_local + (sqrt(1.0 + spec(1)%part(ip,4)**2 + &
                                              spec(1)%part(ip,5)**2 + &
-                                             spec(1)%part(ip,6)**2 )-mu_gamma(1))**2
+                                             spec(1)%part(ip,6)**2 )-mu_gamma(1))**2 &
+                                             *charge(1)
```
Comment (cenit) [inline src/pdf_moments.f90]: here it is the real change!

File: src/read_input.f90 [hunk 5]
```diff
@@ -580,7 +580,7 @@ end subroutine read_input_data
  subroutine consistency_check_number_of_particles_comp
  if(all(ppc>=1)) then
   call from_ppc_to_npx_npy_npz
- elseif(all(np_per_zc)==-1) then
+ elseif(all(np_per_zc==-1)) then
```
Comment (cenit) [inline src/read_input.f90]: again, we are already with this fix in master. Do you have any idea why it is highlighted?

File: src/util.f90 (hunk 2 above)
Comment (albz) [inline src/util.f90]: Sounds appropriate, should we do that? At the end 4 is the best choice, we can now leave like this.

File: src/pdf_moments.f90 (hunk 3 above)
Comment (albz) [inline src/pdf_moments.f90]: Yes, this is just pure formatting

File: src/pdf_moments.f90 (hunk 4 above)
Comment (albz) [inline src/pdf_moments.f90]: That's the change! 👍🏻

File: src/util.f90 (hunk 2 above)
Comment (cenit) [inline src/util.f90]: Just don't change it anymore upstream if you think this is the best choice and you don't want to move it into an input parameter

Comment (albz) [inline src/util.f90]: Yes @cenit, this is what will certainly happen. We had to perform some tests before making sure for the best choice. Please let's keep 4 from now on. If different choices will be demonstrated to be equally important, we will put a control from nml.

**Pull request merged**